examples/python/easy_start_demo/demo_grpc_pbs/demo_pb2_grpc.py

```diff
@@ -0,0 +1,106 @@
+# Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
+import grpc
+
+import demo_pb2 as demo__pb2
+
+
+class GRPCDemoStub(object):
+  """service是用来给GRPC服务定义方法的, 格式固定, 类似于Golang中定义一个接口
+  `service` is used to define methods for GRPC services in a fixed format, similar to defining an interface in Golang
+  """
+
+  def __init__(self, channel):
+    """Constructor.
+
+    Args:
+      channel: A grpc.Channel.
+    """
+    self.SimpleMethod = channel.unary_unary(
+        '/demo.GRPCDemo/SimpleMethod',
+        request_serializer=demo__pb2.Request.SerializeToString,
+        response_deserializer=demo__pb2.Response.FromString,
+        )
+    self.ClientStreamingMethod = channel.stream_unary(
+        '/demo.GRPCDemo/ClientStreamingMethod',
+        request_serializer=demo__pb2.Request.SerializeToString,
+        response_deserializer=demo__pb2.Response.FromString,
+        )
+    self.ServerStreamingMethod = channel.unary_stream(
+        '/demo.GRPCDemo/ServerStreamingMethod',
+        request_serializer=demo__pb2.Request.SerializeToString,
+        response_deserializer=demo__pb2.Response.FromString,
+        )
+    self.BidirectionalStreamingMethod = channel.stream_stream(
+        '/demo.GRPCDemo/BidirectionalStreamingMethod',
+        request_serializer=demo__pb2.Request.SerializeToString,
+        response_deserializer=demo__pb2.Response.FromString,
+        )
+
+
+class GRPCDemoServicer(object):
+  """service是用来给GRPC服务定义方法的, 格式固定, 类似于Golang中定义一个接口
+  `service` is used to define methods for GRPC services in a fixed format, similar to defining an interface in Golang
+  """
+
+  def SimpleMethod(self, request, context):
+    """简单模式
+    Simple
```

Comment:
The term is "unary" or "unary-unary" (python-specific).

Reply:
I will fix the all like this